android/.project

```diff
@@ -1,20 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Have you tested that this gets correctly re-generated?

Reply:
This wasn't intentional, I had meant to revert this.  Will remove.